modules/rpm-ostree/module.yml

```diff
@@ -5,6 +5,7 @@ example: |
   repos:
     - https://copr.fedorainfracloud.org/coprs/atim/starship/repo/fedora-%OS_VERSION%/atim-starship-fedora-%OS_VERSION%.repo # when including COPR repos, use the %OS_VERSION% magic string
     - https://brave-browser-rpm-release.s3.brave.com/brave-browser.repo
+    - my-local-repository.repo # local repo located in `files/rpm-ostree/my-local-repository.repo`
```

Comment:
IMO no need to introduce non-runnable example if not necessary. The example doesn't need to showcase all features either if those are documented elsewhere.
```suggestion
```